src/covid19sim/simulator.py

```diff
@@ -778,11 +854,27 @@ def compute_covid_properties(self):
           self.viral_load_plateau_start, \
             self.viral_load_plateau_end, \
               self.viral_load_recovered = _sample_viral_load_piecewise(
-                                             rng=self.rng, age=self.age,
-                                             initial_viral_load=self.initial_viral_load)
-        self.infectiousness_onset_days = 1 + self.rng.normal(loc=INFECTIOUSNESS_ONSET_DAYS_AVG, scale=INFECTIOUSNESS_ONSET_DAYS_STD)
+                  rng=self.rng,
+                  age=self.age,
+                  initial_viral_load=self.initial_viral_load,
+                  conf=self.conf
+            )
+        self.infectiousness_onset_days = (
+            1
+            + self.rng.normal(
+                loc=self.conf.get("INFECTIOUSNESS_ONSET_DAYS_AVG"),
+                scale=self.conf.get("INFECTIOUSNESS_ONSET_DAYS_STD")
+            )
+        )
         # FIXME : Make incubation_days as lognormal
-        self.incubation_days = self.infectiousness_onset_days + self.viral_load_plateau_start + self.rng.normal(loc=SYMPTOM_ONSET_WRT_VIRAL_LOAD_PEAK_AVG, scale=SYMPTOM_ONSET_WRT_VIRAL_LOAD_PEAK_STD)
+        self.incubation_days = (
+            self.infectiousness_onset_days
+            + self.viral_load_plateau_start
+            + self.rng.normal(
+                loc=self.conf.get("SYMPTOM_ONSET_WRT_VIRAL_LOAD_PEAK_AVG"),
+                scale=self.conf.get("SYMPTOM_ONSET_WRT_VIRAL_LOAD_PEAK_STD")
+            )
+        )
```

Comment:
Are we moving to black format?

Reply:
I did not run `black` specifically on those file (I did on `run.py` `test_hydra.py` and `test_covid_testing.py`) but I did from time to time split up long lines for readability

Reply:
💯